### PR TITLE
update of routes.rb

### PR DESCRIPTION
**Pull request by wozane**
- changed the file context
- connect #32

### CODE REVIEW

**Comment by CallMeSandra** [comment]
:+1:

File: config/routes.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 Rails.application.routes.draw do
   resources :recipes, only: [:index, :show]
-
-  get 'ingredients/index'
+  resources :ingredients, only: [:index]
 
   root 'ingredients#index'
```
Comment (gosiamaks) [inline config/routes.rb]: 'end'  missing

Comment (gotar) [inline config/routes.rb]: end is hidden

**Comment by gotar** [comment]
You should drop those line: `get '/' => 'application#index'`

**Comment by gotar** [comment]
it's not needed anymore you have other route to point to root

**Comment by gotar** [comment]
ok you can merge (squash)

**Comment by wozane** [comment]
connect #12

**Pull request merged**